util/download_data.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const _ = require('lodash');
+const async = require('async');
+const child_process = require('child_process');
+const fs = require('fs-extra');
+const config = require( 'pelias-config' ).generate(require('../schema'));
+const logger = require('pelias-logger').get('download');
+
+if (require.main === module) {
+  download((err) => {
+    if (err) {
+      logger.error('Failed to download data', err);
+      process.exit(1);
+    }
+    logger.info('All done!');
+  });
+}
+
+function download(callback) {
+  let sources;
+
+  if (!_.isEmpty(config.imports.openstreetmap.download)) {
+    sources = _.map(config.imports.openstreetmap.download, (source) => source.sourceURL);
+  }
+  else {
+    sources = [
+      'http://planet.openstreetmap.org/pbf/planet-latest.osm.pbf'
```

Comment:
Yeah, I struggled with this a bit too, but ultimately the entire planet IS the default. :) Hopefully, people can rely on the docs to keep them out of these potential pitfalls. Also, this can always be updated in the future if we get feedback from the community.